packages/plugin/src/lib/content.ts

```diff
@@ -14,6 +14,8 @@ import { crossFetch } from './query'
 import { dbApi } from './db'
 import { chunk } from 'lodash-es'
 import { getSettings } from './settings'
+import * as localModlistSubscriptions from '$lib/localModlistSubscriptions'
+import { ModListIdsResponse } from 'node_modules/@mass-block-twitter/server/src/routes/modlists'
```

Comment:
The reference path is incorrect, only `import {} from '@mass-block-twitter/server'` is allowed. If an export is missing, please modify _packages/server/src/lib.d.ts_.